lms/static/js/spec/student_account/multiple_enterprise_spec.js

```diff
@@ -0,0 +1,74 @@
+define(['edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers', 'js/student_account/multiple_enterprise'],
+    function(AjaxHelpers, MultipleEnterpriseInterface) {
+        'use strict';
+
+        describe('MultipleEnterpriseInterface', function() {
+            var LEARNER_URL = '/enterprise/api/v1/enterprise-learner/?username=test-learner',
+                NEXT_URL = '/',
+                REDIRECT_URL = '/enterprise/api/v1/multiple-enterprise/';
```

Comment:
nit: can we set the url to `/enterprise/select/active` as implemented in https://github.com/edx/edx-enterprise/pull/609 ?

Reply:
Done.